src/main/kotlin/io/github/gradlenexus/publishplugin/NexusHost.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.gradlenexus.publishplugin
+
+import java.net.URI
+
+enum class NexusHost(private val baseUrl: String) {
```

Comment:
`NexusHost` is a too wide naming. `Nexus` can be hosted by anyone, so I think `NexusHost` is a wrong name.

What do you think of `OssrhHost` or `SonatypeOssrhHost`?

Reply:
Good call! I'd prefer `SonatypeOssrhHost`.

Reply:
On the other hand, this could be used for other, non-Sonatype OSSRH hosts in the future. 🤔

Reply:
Since we're going to make it configureable for the user, I'll keep the name as `NexusHost`.